mesh/nurbs.hpp

```diff
@@ -67,6 +67,9 @@ class KnotVector
    void CalcD2Shape(Vector &grad2, int i, double xi) const
    { CalcDnShape(grad2, 2, i, xi); }
 
+   void FindMaxima(Array<int> &ks,Vector &xi,Vector &u);
```

Comment:
Most functions in this class have no documentation, but it would be helpful to add documentation at least to the new functions.

Reply:
Documentation added in [0313c71](https://github.com/mfem/mfem/pull/3232/commits/0313c718d871bc4b3b8803db162d3de018859843)